.github/workflows/main.yml

```diff
@@ -0,0 +1,85 @@
+name: CI - Integration test
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+# A workflow run is made up of one or more jobs that can run sequantially or in parallel
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    environment: dev
+    strategy:
+      matrix:
+        postgres-version: [12]
+        python-version: [3.8]
+        node-version: [14]
+
+    services:
+      postgres:
```

Comment:
What's the reason why postgres is added through a `service` and not a `step`?

Reply:
Good question, we could easily run the postgres db with a docker command, but since GitHub recommended to use postgres as a service container, I thought we should use this way (https://docs.github.com/en/actions/using-containerized-services/creating-postgresql-service-containers)